spec/20-http_request_header_format.md

```diff
@@ -176,6 +174,28 @@ There are two additional options that vendors MAY follow:
 - A component that makes a deferred or delayed recording decision may communicate the priority of a recording by setting `sampled` flag to `1` for a subset of requests.
 - A component may also fall back to probability sampling and set the `sampled` flag to `1` for the subset of requests.
 
+##### Random Trace ID Flag
+
+<!--
+
+TODO: how many random bytes are needed?
+7 was chosen as it can be efficiently represented as a 64-bit signed or unsigned integer.
+8 would require an unsigned long which is not supported by some languages (like Java and Go).
+63 bits would be possible, but would require a more complex description that may be more difficult to understand.
+
+TODO: Which specific bytes should be random?
+The right-most bytes were chosen because some tracing systems are known to use the left-most
+portion of the trace id for non-random data such as a timestamp component.
+
+TODO: Do we want to place any restrictions on the randomness or is saying "MUST be random" enough?
+As an example, RFC4122 (UUID v4) simply states "Set all the other bits to randomly (or pseudo-randomly) chosen values."
+https://datatracker.ietf.org/doc/html/rfc4122#section-4.4
+
+-->
+
+When set, the second least significant bit (second from the right), denotes that the right-most 7 bytes of the trace ID MUST be random (or pseudo-random).
```

Comment:
*nitpick:* I know this is the same phrasing that we use for `sampled`, but when actually reading it, I find the phrasing a bit awkward. 

How about something like this: 

> The second least significant bit of the `trace-flags` field denotes the `random-trace-id` flag. If that flag is set, the right-most 7 bytes of the trace ID MUST be random (or pseudo-random).

Additionally, this section currently does not introduce the spelling `random-trace-id` (lower case, kebap case) but we refer to it by that spelling later on. Thus having that exact spelling here in the definition might be good, I think.